src/tools/compiletest/src/runtest.rs

```diff
@@ -82,21 +82,21 @@ fn disable_error_reporting<F: FnOnce() -> R, R>(f: F) -> R {
 }
 
 /// The platform-specific library name
-pub fn get_lib_name(lib: &str, dylib: bool) -> String {
-    // In some casess (e.g. MUSL), we build a static
-    // library, rather than a dynamic library.
-    // In this case, the only path we can pass
-    // with '--extern-meta' is the '.lib' file
-    if !dylib {
-        return format!("lib{}.rlib", lib);
-    }
-
-    if cfg!(windows) {
-        format!("{}.dll", lib)
-    } else if cfg!(target_os = "macos") {
-        format!("lib{}.dylib", lib)
-    } else {
-        format!("lib{}.so", lib)
+fn get_lib_name(lib: &str, aux_type: AuxType) -> Option<String> {
+    match aux_type {
+        AuxType::Bin => None,
+        // In some casess (e.g. MUSL), we build a static
```

Comment:
```suggestion
        // In some cases (e.g. MUSL), we build a static
```

Reply:
That typo is old and I don't want to slip back in the homu queue :/

Can we leave it please?

Reply:
I can quickly re-approve after the change. The queue position doesn't depend on the time when a PR has been approved. It depends on when it was opened, whether it was marked to always be included in a rollup and any explicitly set priority.